kmk/modules/combos.py

```diff
@@ -171,7 +175,6 @@ def on_release(self, keyboard, key, int_coord):
 
                 key = combo.result
                 break
-
```

Comment:
Please leave the line break. Visual separation of functional blocks in the code aids readability.